### PR TITLE
Support CentOS/RHEL installation

### PR DESCRIPTION
**Pull request by fivaldi**
eduroam.olomouc.ueb.cas.cz - test pilot

### CODE REVIEW

**Comment by fivaldi** [comment]
@semik Just a note: I'd suggest verifying this on some Debian box using dry-run, checking diff for undesired effects. Do you have any candidate(s), pls?

File: tasks/main.yml [hunk 1]
```diff
@@ -138,16 +200,15 @@
 
 - name: PKCS#12 certs
   include_tasks: pkcs12_cert.yml
-  when: ((eduroam.radsec is defined) and (eduroam.radsec.certificate is undefined)) and
-        ((eduroam.EAP is defined) and (eduroam.EAP.certificate is undefined))
+  when: eduroam.radsec is undefined or eduroam.EAP is undefined
```
Comment (semik) [inline tasks/main.yml:203]: Interesting condition, it took me quite a time to understand it. Ive created issue from it https://github.com/CESNET/ansible-freeradius/issues/17

File: tasks/main.yml [hunk 2]
```diff
@@ -138,16 +200,15 @@
...
 
 - name: PEM certs for EAP
   include_tasks: pem_cert_eap.yml
-  when: (eduroam.EAP is defined) and (eduroam.EAP.certificate is defined)
+  when: eduroam.EAP is defined and eduroam.EAP.certificate is defined and eduroam.EAP.private_key is defined
```
Comment (semik) [inline tasks/main.yml:207]: I'm not lazy to write parenthesis, I'm lazy/dumb to learn about operator priority. But accepting anyway.

File: tasks/main.yml [hunk 3]
```diff
@@ -138,16 +200,15 @@
...
 
 - name: PEM certs for RadSec
   include_tasks: pem_cert_radsec.yml
-  when: eduroam.radsec.certificate is defined
+  when: eduroam.radsec is defined and eduroam.radsec.certificate is defined and eduroam.radsec.private_key is defined
```
Comment (semik) [inline tasks/main.yml:211]: Good catch!

File: tasks/main.yml [hunk 4]
```diff
@@ -10,36 +10,92 @@
     #update_cache: yes
   when: ansible_distribution_release=='jessie'
 
-- name: Update repo cache
+- name: Set facts depending on OS family (Debian)
+  set_fact:
+    etc_freeradius: /etc/freeradius
```
Comment (semik) [inline tasks/main.yml:15]: In future /etc/freeradius/3.0 at least for Stretch & Buster.

Comment (semik) [reply tasks/main.yml:15]: and on line 103 revert to /etc/freeradios only on Jessie

**Review by semik — APPROVED** [review]

File: tasks/main.yml (hunk 2 above)
Comment (fivaldi) [reply tasks/main.yml:207]: Thank you.

File: tasks/main.yml (hunk 3 above)
Comment (fivaldi) [reply tasks/main.yml:211]: Thanks.

File: tasks/main.yml (hunk 4 above)
Comment (fivaldi) [reply tasks/main.yml:15]: Thanks for pointing this out. Created  #18

**Comment by fivaldi** [comment]
Thanks for review!

**Pull request merged**